Assignment1/src/main/java/models/Deck.java

```diff
@@ -9,21 +9,52 @@
  * Deck Class that uses the Card class to maintain a sinlge unique deck.
  */
 public class Deck {
-    private ArrayList<Card> deckList;
+    private static int NUM_SIZE = 52;
+    public Card cards[];
+    public int count;
+
+    public ArrayList<Card> deckList;
     /* Creating a deckList that holds the array of cards*/
-    public Deck() {
-        deckList = new ArrayList<>();
-    }
+
     /* Grab the cards from array*/
     public ArrayList<Card> getCards(){
         return deckList;
```

Comment:
It's an inadvertent line space, not a huge deal. Something git just picks up on.